akka-javasdk/src/main/java/akka/javasdk/prompt/PromptTemplate.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.prompt;
+
+import akka.Done;
+import akka.javasdk.annotations.ComponentId;
+import akka.javasdk.eventsourcedentity.EventSourcedEntity;
+
+import java.util.Optional;
+
+import static akka.Done.done;
+
+@ComponentId("akka-prompt-template")
```

Comment:
I'm open to any naming suggestions for:
 - componentId
 - component class name
 - component state class name
 - and others :)

Reply:
I'm wondering if there is a good reason to have all these new AI-related abstractions under e.g. `akka.javasdk.ai`.. the prompt, the agent, etc.

Reply:
I think we can place this in the same package as the agent, i.e. `agent`.